data/json/professions.json

```diff
@@ -68,7 +71,8 @@
                 "boots_winter",
                 "knit_scarf",
                 "pockknife",
-                "water_clean",
+                "energy_drink",
+                "chips3",
```

Comment:
Yeah, energy drink + chips evokes image of a dorito + dr. peppers drinking slob.
Rest of the equipment is generally more survivor stuff - not everyone carries a pocket knife and winter gear (not even in winter).